<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Image1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>42</y>
     <width>112</width>
     <height>112</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Image2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>42</y>
     <width>112</width>
     <height>112</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;Noto Sans CJK KR Bold&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Image3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>178</y>
     <width>112</width>
     <height>112</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Image4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>178</y>
     <width>112</width>
     <height>112</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Image8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>112</width>
     <height>112</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Image7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>112</width>
     <height>112</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Image5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>314</y>
     <width>112</width>
     <height>112</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Image6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>314</y>
     <width>112</width>
     <height>112</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line1">
   <property name="windowModality">
    <enum>Qt::NonModal</enum>
   </property>
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>90</y>
     <width>78</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="midLineWidth">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line2">
   <property name="windowModality">
    <enum>Qt::NonModal</enum>
   </property>
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>230</y>
     <width>78</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="midLineWidth">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line3">
   <property name="windowModality">
    <enum>Qt::NonModal</enum>
   </property>
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>360</y>
     <width>78</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="midLineWidth">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line4">
   <property name="windowModality">
    <enum>Qt::NonModal</enum>
   </property>
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>499</y>
     <width>78</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="midLineWidth">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_predict_value_1">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>92</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>0.0000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_predict_value_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>226</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>0.0000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_predict_value_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>362</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_predict_value_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>498</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_selectImage1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>154</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;이미지 선택&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>선택</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deleteImage1">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>154</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_selectImage2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>154</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>선택</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deleteImage2">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>154</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_selectImage4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>선택</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deleteImage4">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>290</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_selectImage3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>선택</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deleteImage3">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>290</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deleteImage6">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>426</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_selectImage6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>426</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>선택</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_selectImage5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>426</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>선택</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deleteImage5">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>426</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_selectImage7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>562</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>선택</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deleteImage8">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>562</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deleteImage7">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>562</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_selectImage8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>562</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>선택</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_predict_1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>82</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;단일 예측 값 출력&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Predict</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_predict_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>354</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Predict</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_predict_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>218</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Predict</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_predict_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>490</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Predict</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Load_single_model">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Model</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>197</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_1">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;CheckBox : predict 버튼을 눌러야 활성화 됩니다.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;멀티모달 분석을 원하는 Set를 선택하세요&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>331</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>469</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mm_predict">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>310</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;Print predicted value of several set of characters&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 26pt &quot;Noto Sans CJK KR Medium&quot;;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_svm_1">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>420</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Predict</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_quit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>프로그램 종료</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>50</y>
     <width>55</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Set 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>190</y>
     <width>55</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Set 2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>320</y>
     <width>55</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Set 3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>460</y>
     <width>55</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Set 4&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_predictList">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Noto Sans CJK KR Black&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>set 1 predict value :0.0000
Set 2 predict value :0.0000
Set 3 predict value :0.0000
Set 4 predict value :0.0000</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_predictList_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 13pt &quot;Noto Sans CJK KR Bold&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>체크 된 멀티 모달 예측 값 :</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_option">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;이미지 전처리 먼저 진행(이진화, crop&amp;amp;resize)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>image processing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear_checkbox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>540</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear CheckBox</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
